pkg/common/apis/header.go

```diff
@@ -56,6 +56,16 @@ func HTTPLanguage(r *http.Request) i18n.LanguageCodes {
 	return langs
 }
 
+// GetApplicationId .
+func GetApplicationId(ctx context.Context) string {
```

Comment:
don't put this here. It's not common.
`applicationId` or `applicationID` or `application-id` ?

Reply:
Have moved this to release